app/controllers/placements/schools/placements/build_controller.rb

```diff
@@ -25,8 +33,10 @@ def add_mentors
   end
 
   def check_your_answers
-    @placement = initialize_placement
+    @placement = initialize_placement.decorate
```

Comment:
Hooray 🙌🏻  you got the placement decorator working on `Placements::Schools::Placements::Build::Placement` instances!